app/assets/javascripts/discourse/app/helpers/category-color-variable.js

```diff
@@ -0,0 +1,3 @@
+import { htmlHelper } from "discourse-common/lib/helpers";
+
+export default htmlHelper((color) => `--category-color: #${color}`);
```

Comment:
Just in case anyone changes the order of the styles or adds a new one:

```suggestion
export default htmlHelper((color) => `--category-color: #${color};`);
```